slice/Ice/Ice2Definitions.ice

```diff
@@ -28,6 +28,12 @@ module Ice
     {
     }
 
+    /// The keys for supported Ice2 connection parameters.
+    unchecked enum Ice2ParameterKey : int
+    {
+        OutgoingFrameMaxSize = 0
```

Comment:
It's not obvious from the name direction of the connection this applies to. Does it apply to both client and server? Is there any negotiation between the client and server's OutgoingFrameMaxSize?

Reply:
It's renamed IncomngFrameMaxSize now.